cron/main.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"sort"
+	"time"
+
+	"github.com/jszwec/csvutil"
+)
+
+type Repository struct {
+	Repo     string `csv:"repo"`
+	Metadata string `csv:"metadata,omitempty"`
+}
+
+func main() {
+	fileName := fmt.Sprintf("%d-%d-%d.json", time.Now().Month(), time.Now().Day(), time.Now().Year())
+	result, err := os.Create(fileName)
+	if err != nil {
+		panic(err)
+	}
+	defer result.Close()
+	projects, err := os.OpenFile(os.Args[1], os.O_RDONLY, 0644)
+	if err != nil {
+		panic(err)
+	}
+	defer projects.Close()
+
+	repos := []Repository{}
+	data, err := ioutil.ReadAll(projects)
+	if err != nil {
+		panic(err)
+	}
+	err = csvutil.Unmarshal(data, &repos)
+	if err != nil {
+		panic(err)
+	}
+	sort.Slice(repos, func(i, j int) bool {
+		return repos[i].Repo < repos[j].Repo
+	})
+
+	//nolint
+	const checks string = "--checks=Active,CI-Tests,CII-Best-Practices,Code-Review,Contributors,Frozen-Deps,Fuzzing,Packaging,Pull-Requests,SAST,Security-Policy,Signed-Releases,Signed-Tags"
+
+	if _, err = result.Write([]byte(`{"results":[`)); err != nil {
+		panic(err)
+	}
+	for _, r := range repos {
+		fmt.Println(r.Repo)
+		//nolint
+		cmd := exec.Command("../scorecard", fmt.Sprintf("--repo=%s", r.Repo), fmt.Sprintf("--metadata=%s", r.Metadata), checks, "--show-details", "--format=json")
```

Comment:
Can we call some function (or perform some refactoring) within the scorecard module, rather than invoke the binary?

Reply:
Yes, I agree. But it is a larger refactor. Lets do that later.

Reply:
SGTM.